css/main.css

```diff
@@ -307,21 +307,134 @@ li {
 }
 
 .list-title {
-  font-family: "Inter", sans-serif;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: bold;
   font-size: 32px;
   line-height: 44px;
 }
 
 .languages {
-  list-style: inside url("../media/svg/rectangle.svg");
+  list-style: inside url('../media/svg/rectangle.svg');
 }
 
 .frameworks {
-  list-style: inside url("../media/svg/square.svg");
+  list-style: inside url('../media/svg/square.svg');
 }
 
 .skills {
-  list-style: inside url("../media/svg/circle.svg");
+  list-style: inside url('../media/svg/circle.svg');
+}
+
+.contact-section {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  margin-bottom: 50px;
+}
+
+.contact-section-head {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.contact-section-info {
+  margin-top: 50px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 44px;
+  font-family: Arial, Helvetica, sans-serif;
+  text-align: center;
+  width: 327px;
+  color: #172b4d;
+}
```

Comment:
- [ ] Kindly make sure to leave enough space between the contact information section and the contact form.

    - Your Design
    
  ![image](https://user-images.githubusercontent.com/87197412/184039328-d782b60b-4742-49dc-b900-8a50c971a9a8.png)

    - Figma Design
     
  ![image](https://user-images.githubusercontent.com/87197412/184039286-a96c6790-d89e-4df4-aa24-4aef7acc63f8.png)